src/01-general/document-templates.apib

```diff
@@ -38,3 +38,4 @@ Get a list of all document templates.
                         + timetracking_report
                         + workorder
                 + is_default: true (boolean)
+                + name: `new logo` (string)
```

Comment:
B: this should be a nullable string. We don't have a name for the default variations. We should return null in this case